packages/cloud-cognitive/src/components/PageHeader/PageHeader.js

```diff
@@ -12,6 +12,7 @@ import cx from 'classnames';
 import { useResizeDetector } from 'react-resize-detector';
 
 import { Grid, Column, Row, Button, Tag } from 'carbon-components-react';
+import { breakpoints } from '@carbon/layout/es';
```

Comment:
```suggestion
import { breakpoints } from '@carbon/layout';
```

Was there a reason for specifically loading from /es? The problem is that jest wants the /lib while browser builds typically want the /es, so explicitly coding one of the other won't work in both places. That's why the tests are failing. There ought to be a top-level export that the package.json ensures is loaded in the form that's required so that we don't need the /es, but I've not tested it!

Reply:
Yes, in error.